lib/puppet/reference/configuration.rb

```diff
@@ -54,6 +54,13 @@
 
 * Each of these settings can be specified in `puppet.conf` or on the
   command line.
+* Puppet Enterprise (PE) shares configuration settings used in open source
+  Puppet and documented in the configuration reference. However, PE defaults
```

Comment:
"`Puppet Enterprise (PE) shares configuration settings used in open source Puppet and documented in the configuration reference.`" feels like there's a verb missing before "`documented`". Should that be "`... used in open source Puppet and is documented in the configuration reference`"? Should "`documented`" be "`documentation`" instead (shares configuration settings and documentation)?

Reply:
The two clauses are "used in open source Puppet" and "documented in the configuration reference".

Puppet Enterprise (PE) shares configuration settings / used in open source Puppet

Puppet Enterprise (PE) shares configuration settings / documented in the configuration reference

Reply:
Jorie asked me to chime in. How about: "Puppet Enterprise (PE) and open source Puppet share the configuration settings that are documented here. However, PE defaults for some settings differ from the Puppet defaults." etc.

Reply:
This had completely fallen off my radar.

@kayayarai 👍 That completely addresses the grammar concerns that I had.

Reply:
@kayayarai @jhelwig @jtappa I've revised the statement.